cocos/network/WebSocket.h

```diff
@@ -238,7 +239,7 @@ class CC_DLL WebSocket
 
     struct lws*         _wsInstance;
     struct lws_context* _wsContext;
-    std::shared_ptr<bool> _isDestroyed;
+    std::shared_ptr<std::atomic<bool>> _isDestroyed;
```

Comment:
Yep, i see.
What i mean is why we need std::shared_ptr<std::atomic<bool>> instead of std::atomic<bool>, because bool is a raw type.